### PR TITLE
List Andromeda $ANDR

### PR DESCRIPTION
**Pull request by SvansyVonSwansea**
Addition of Andromeda, $ANDR

### Adding Assets

If adding a new asset, please ensure the following:
- [x] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
- [x] Add asset to bottom of `zone_assets.json`.
   - [x] `chain_name` and `base_denom` are provided and use values exactly as defined at the Chain Registry.
   - [x] `path` is provided, and the IBC channel referenced is registered at the Chain Registry (unless native to Osmosis).
   - [x] `osmosis_verified` is set to `false`
   - [x] `listing_date_time_utc` is specified and accurate
   - [x] Optional: `transfer_methods`, `peg_mechanism`, `override_properties`, `canonical`, `categories`, where necessary (see [README](https://github.com/osmosis-labs/assetlists/tree/main?tab=readme-ov-file#how-to-add-assets) for details).

### Adding Chains

If adding a new chain, please ensure the following:
- [x] Chain is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - Chain's registration must have `staking` defined, with at least one `staking_token` denom specified.
   - Chain's registration must have `fees` defined; at least one fee token has low, average, and high gas prices defined.
- [x] IBC Connection between chain and Osmosis is registered.
- [x] Add chain to bottom of `zone_chains.json`
   - [ ] `rpc` and `rest` does not have any CORS blocking of the Osmosis domain, and RPC node has have WSS enabled.
   - [ ] `explorer_tx_url` correctly directs to the transaction when the hash is inserted into the URL.

### CODE REVIEW

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -5077,6 +5077,16 @@
...
+      "base_denom": "ibc/631DB9935E8523BDCF76B55129F5238A14C809CCB3B43AECC157DC19702F3F9E",
+      "path": "transfer/channel-81924",
+      "osmosis_verified": false,
+      "chain_name": "andromeda",
+      "base_denom": "uandr"
+       "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/andromeda/images/andromeda-logo.png"
       }
     }
   ]
```
Comment (Cordtus) [inline osmosis-1/osmosis.zone_assets.json:5092]: The red thing means bad

**Review by SvansyVonSwansea — COMMENTED** [review]
fixed error

**Review by SvansyVonSwansea — COMMENTED** [review]
fixed error

**Review by SvansyVonSwansea — COMMENTED** [review]
format fix 😬

**Review by SvansyVonSwansea — COMMENTED** [review]
promise last fix 😭

**Review by SvansyVonSwansea — COMMENTED** [review]
this should work 🤔💭

**Comment by JeremyParish69** [comment]
Please see the checklist. The chain must also be added to zone_chains

**Comment by SvansyVonSwansea** [comment]
> Please see the checklist. The chain must also be added to zone_chains

thanks. it was added to zone.chains now as well.

**Review by SvansyVonSwansea — COMMENTED** [review]
added info from zone_chains

(something went wrong, fix coming)

**Comment by JeremyParish69** [comment]
> thanks. it was added to zone.chains now as well.

zone_chains still has not been modified in this PR.

**Comment by SvansyVonSwansea** [comment]
> > thanks. it was added to zone.chains now as well.
> 
> zone_chains still has not been modified in this PR.

sorry misread. added now.

**Comment by JeremyParish69** [comment]
> sorry misread. added now.

No you're still not following direction. You keep creating new PRs. It must be added to this PR

**Comment by SvansyVonSwansea** [comment]
> > sorry misread. added now.
> 
> No you're still not following direction. You keep creating new PRs. It must be added to this PR

sorry. i think i did it correctly this time?

**Comment by JeremyParish69** [comment]
> sorry. i think i did it correctly this time?

Yes, it has been added to the zone_chains file now. However the PR still needs a correct title, the asset addition still needs correcting, the conflicts still needs to be resolved, and the checklist needs to be ticked.

**Pull request merged**